### PR TITLE
Removing extraneous columns from demographic table

### PR DESCRIPTION
**Pull request by achasmita**

### CODE REVIEW

**Comment by shankari** [comment]
@achasmita
1. please indicate testing done. "If it is not tested, it is broken"
2. From a clean commit history perspective, the second commit message is a duplicate of the first and is incorrect
3. how come these didn't show up in the testing of the previous PR (#63)? Are these columns not in the test dataset that has been shared with you?

File: utils/db_utils.py [hunk 1]
```diff
@@ -107,13 +107,14 @@ def query_demographics():
...
-    df.columns=[col.rsplit('.',1)[-1] if col.startswith('data.jsonDocResponse.') else col for col in df.columns]  
-    df.drop(columns=['xmlns:jr', 'xmlns:orx', 'id'], inplace = True) 
+        columns_to_drop = [col for col in df.columns if col.startswith("metadata")]
+        df.drop(columns= columns_to_drop, inplace=True) 
+        modified_columns = perm_utils.get_demographic_columns(df.columns)  
+        df.columns = modified_columns 
+        df.columns=[col.rsplit('.',1)[-1] if col.startswith('data.jsonDocResponse.') else col for col in df.columns]  
+        for col in constants.EXCLUDED_DEMOGRAPHICS_COLS:
+            if col in df.columns:
+                df.drop(columns= [col], inplace=True)
```
Comment (shankari) [inline utils/db_utils.py:117]: why isn't this just 

```suggestion

        df.drop(columns=constants.EXCLUDED_DEMOGRAPHICS_COLS, inplace=True)
```

**Comment by achasmita** [comment]
> @achasmita
> 
> 1. please indicate testing done. "If it is not tested, it is broken"
> 2. From a clean commit history perspective, the second commit message is a duplicate of the first and is incorrect
> 3. how come these didn't show up in the testing of the previous PR ([Updated readme and added demographics table #63](https://github.com/e-mission/op-admin-dashboard/pull/63))? Are these columns not in the test dataset that has been shared with you?

No, those columns are not available in the dataset provided to me. So it was all good when I tested locally. But I saw extraneous columns while in staging.

File: utils/db_utils.py (hunk 1 above)
Comment (achasmita) [reply utils/db_utils.py:117]: As this columns are not i the data set I am working, we cannot directly exclude all those columns.

**Comment by shankari** [comment]
Merging this now to try and resolve showstopper
https://github.com/e-mission/op-admin-dashboard/issues/65

**Comment by shankari** [comment]
@achasmita I am squash-merging this, please take it into account while pulling from upstream.

**Pull request merged**